src/view/event-view.js

```diff
@@ -0,0 +1,81 @@
+import { createElement } from '../render.js';
+import { formatDate, humanizeTime, getDurationTime} from '../util.js';
+
+const createEventTemplate = (point) => {
+  const {
+    basePrice,
+    destination,
+    type,
+    dateFrom,
+    dateTo,
+    isFavorite,
+    offers
+  } = point;
+
+  const favoriteClassName = isFavorite ?
+    'event__favorite-btn  event__favorite-btn--active':
+    'event__favorite-btn';
+
+  return (
+    `<li class="trip-events__item">
+      <div class="event">
+      <time class="event__date datetime="${formatDate(dateFrom, 'YYYY-MM-DD')}">${formatDate(dateFrom, 'DD MMM')}</time>
+      <div class="event__type">
+          <img class="event__type-icon" width="42" height="42" src="img/icons/${type}.png" alt="Event type icon">
+      </div>
+      <h3 class="event__title">${type} ${destination.name}</h3>
+      <div class="event__schedule">
+          <p class="event__time">
+          <time class="event__start-time" datetime="${formatDate(dateFrom, 'YYYY-MM-DDTHH:mm')}">${humanizeTime(dateFrom)}</time>
+          &mdash;
+          <time class="event__end-time" datetime="${formatDate(dateTo, 'YYYY-MM-DDTHH:mm')}">${humanizeTime(dateTo)}</time>
+          </p>
+          <p class="event__duration">${getDurationTime(dateFrom, dateTo)}</p>
+      </div>
+      <p class="event__price">
+          &euro;&nbsp;<span class="event__price-value">${basePrice}</span>
+      </p>
+      <h4 class="visually-hidden">Offers:</h4>
+     <ul class="event__selected-offers">
+      ${offers ? offers.map(({title, price}) =>
```

Comment:
Пока можно так вывести
```suggestion
      ${offers?.map(({title, price}) =>
```